proto/eventlog.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package eventlog;
+
+option go_package = "proto;eventlog";
+
+message GetEventLogChunkRequest {
+  // The request context.
+  context.RequestContext request_context = 1;
+
+  // The invocation identifier itself.
+  string invocation_id = 2;
+
+  message OptionalChunklID {
+    int32 value = 1;
+  }
+
+  // Optionally specify the chunk to be returned; gets the last chunk if unset
+  OptionalChunkID chunk_id = 3;
+}
+
+message GetEventLogChunkResponse {
+  // The response context.
+  context.ResponseContext response_context = 1;
+
+  message Chunk {
+    int32 chunk_id = 1;
```

Comment:
see prev and next comment below but making this a string gives you some freedom.

(It's basically a pagination token -- and having those be opaque is quite useful! In the past I've written code where a proto containing {int offset, int limit} is encoded and base64d and then used as a pagination token -- then when the server gets that token it can open it up and know where to resume result pagination from.)